sample.env

```diff
@@ -31,10 +10,13 @@ ADMIN_EMAIL=""
 ADMIN_PASSWORD=""
 
 # DJANGO (see settings.py)
-SECRET_KEY=""
+SECRET_KEY="234fjwefjskldjfoqih3232"
```

Comment:
Are you sure we want to add the secret key to versioning? This would make it public...

Reply:
that's just a dummy value, not the real one we are using.